Monika After Story/game/script-stories.rpy

```diff
@@ -485,6 +485,27 @@ label mas_story_o_tei:
     m 3esa "As she came to, she had no memory of what happened in the inn."
     m 1hua "Not long after, Kenji married her as soon as they could, and lived on happily for the rest of their lives."
     return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_story_crow_and_pitcher",
+        prompt="The Crow and the Pitcher",unlocked=False),code="STY")
+
+label mas_story_crow_and_pitcher:
+    call mas_story_begin
+    m 3esd "It was a spell of hot, dry weather. Not even the birds could find much to drink."

```

Comment:
Spell does make sense here, but I doubt most people have heard of the word used in this situation. I don't really see Monika using the word in this fashion either, but that's just me.

Reply:
This is true, I might replace it with "drought" or something similar.